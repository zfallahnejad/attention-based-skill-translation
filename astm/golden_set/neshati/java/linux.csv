871026,linux
40342,linux
