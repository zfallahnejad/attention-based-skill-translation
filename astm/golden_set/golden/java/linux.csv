871026,13,9,0.5909090909090909
40342,12,7,0.631578947368421
1093528,10,10,0.5
